### DISCUSSION TITLE
3.9.10.19群成员信息中memberNickname是空的中文名称

### DISCUSSION THREAD

**Discussion by 108777771** [ttttupup/wxhelper#435] [Q&A]
{"admin":"xiaohuantongxie","adminNickname":"","chatRoomId":"34871209000@chatroom","memberNickname":"^G^G","members":"wxid_i3hqu98qxdlm32^Gxiaohuantongxie^Glaogu2000"},"msg":"success"}
memberNickname返回信息为^G^G

**Comment by 108777771** [comment]
getContactProfile返回
{"code":200,"data":null,"msg":"Not Implemented"}

**Comment by Chener-03** [comment]
![image](https://github.com/ttttupup/wxhelper/assets/68767963/dfced972-b98e-498d-afe8-c4968c3b917d)

**Comment by estn** [comment]
请问你这是那个接口 可以获取群成员的昵称呢 我看3.9.8.25版本 没有这个功能呢

**Comment by misoinc** [comment]
可以通过wx_id,直接查联系人表，就可以拿到名称了，

**Reply by estn** [reply]
这个只能查询自己的好友 不能查询群成员的昵称哇
